tests/features/custom_page/api.feature

```diff
@@ -6,15 +6,16 @@ Feature: Custom page API
 
   Scenario: Programmatically create a Custom page
     Given the following collection:
-      | title             | Le Foie Heureux |
-      | owner             | Rufus Drumknott |
-      | logo              | logo.png        |
-      | moderation        | yes             |
-      | closed            | yes             |
-      | elibrary creation | facilitators    |
+      | uri               | http://joinup.eu/custom_page_api_collection |
+      | title             | Le Foie Heureux                             |
+      | owner             | Rufus Drumknott                             |
+      | logo              | logo.png                                    |
+      | moderation        | yes                                         |
+      | closed            | yes                                         |
+      | elibrary creation | facilitators                                |
     And custom_page content:
-      | title      | body                                     | groups audience |
-      | Dummy page | This is some dummy content like foo:bar. | Le Foie Heureux |
+      | title      | body                                     | groups audience                             |
+      | Dummy page | This is some dummy content like foo:bar. | http://joinup.eu/custom_page_api_collection |
```

Comment:
:( Doesn't this work any more? It's much more preferable to use the title instead of the URL.